oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/editor/FulltextDocumentMaker.java

```diff
@@ -224,13 +238,19 @@ private boolean indexProperty(String path,
             addBinary(doc, null, binaryValues);
             dirty = true;
         } else {
+            if (property.getType() == Type.STRING || property.getType() == Type.STRINGS) {
```

Comment:
It could be the case that the large property that shows up here might not be indexed as property or fulltext. I think this block should be extracted out in some private method and called individually from code indexing property and/or fulltext (that can lead to double warns one each for prop and fulltext but that might be ok I think).

Reply:
Created a new method and logging on both fulltext as well as property indexing.